site/en/reference/cli/concepts/netdata.md

```diff
@@ -189,6 +223,25 @@ the complete Active Operational Dataset.
     Done
     ```
 
+## Debugging & diagnostics
+
+### `full` commands
```

Comment:
Can you redo this section to be focused what a user would debug?  Otherwise this just repeats what's added in the command section.

So one scenario is monitoring whether Thread Network Data is full or not.  Abtin provided a bunch of info in his previous comment that we should add here... i.e. this is why it might get full, how to tell when it's full, and what to do in that scenario.